### ISSUE TITLE
Warning when plotting feh with log scale.

### ISSUE THREAD

**Issue by jason-neal** [DanielAndreasen/SWEETer-Cat#48]
```
/home/jneal/Projects/SWEETer-Cat/sweetercat/plot.py:192: RuntimeWarning: invalid value encountered in log10
  yh1, yh2 = np.log10(min(y)), np.log10(max(y))
/home/jneal/miniconda3/envs/sweetcat/lib/python3.6/site-packages/numpy/lib/function_base.py:786: RuntimeWarning: invalid value encountered in less
  if (np.diff(bins) < 0).any():

```
And all the values on the plot is not shown.

**Comment by jason-neal** [comment]
A test to bring the warning up, but it is more noticable when looking at page.
```
def test_feh_with_log_scale(client):
    test_data = {'color': 'Blue', 'x': "teff", 'y': "feh", 'z': 'Vmag',
                 'x1': "None", 'x2': "None", 'y1': "None", 'y2': "None",
                 'xscale': 'linear', 'yscale': 'log', 'checkboxes': ''}

    for end_point in ('plot',):  # 'plot_exo'
        plot = client.post(url_for(end_point), data=test_data, follow_redirects=True)
        assert plot.status_code == 200
```

**Comment by DanielAndreasen** [comment]
Nice catch. This should only be a problem for `feh` because of the negative values. I can't see any other ways than to set the scale to linear if `feh` is selected. Can maybe make it a bit more generic and set it to linear if there are negative values.

**Comment by DanielAndreasen** [comment]
This has been fixed in commit eb73cfa18ea183b5fe4f on branch [tests](https://github.com/DanielAndreasen/SWEETer-Cat/tree/tests).
A message is flashed when the scale is changed compared to what the user want (set scale to `log`  for `feh` and `submit` to see this effect).

**Comment by jason-neal** [comment]
Very nice. It seems to work well.
